services/checks/checks.go

```diff
@@ -375,29 +445,45 @@ func (s *Service) minPMMAgentVersion(t check.Type) *version.Parsed {
 }
 
 // executeChecks runs all available checks for all reachable services.
-func (s *Service) executeChecks(ctx context.Context) {
+func (s *Service) executeChecks(ctx context.Context) error {
 	s.l.Info("Executing checks...")
 
+	disabledChecks, err := s.GetDisabledChecks()
+	if err != nil {
+		return err
+	}
+
 	var checkResults []sttCheckResult
 
-	mySQLCheckResults := s.executeMySQLChecks(ctx)
+	mySQLCheckResults := s.executeMySQLChecks(ctx, disabledChecks)
 	checkResults = append(checkResults, mySQLCheckResults...)
 
-	postgreSQLCheckResults := s.executePostgreSQLChecks(ctx)
+	postgreSQLCheckResults := s.executePostgreSQLChecks(ctx, disabledChecks)
 	checkResults = append(checkResults, postgreSQLCheckResults...)
 
-	mongoDBCheckResults := s.executeMongoDBChecks(ctx)
+	mongoDBCheckResults := s.executeMongoDBChecks(ctx, disabledChecks)
 	checkResults = append(checkResults, mongoDBCheckResults...)
 
 	s.alertsRegistry.set(checkResults)
+
+	return nil
 }
 
 // executeMySQLChecks runs MySQL checks for available MySQL services.
-func (s *Service) executeMySQLChecks(ctx context.Context) []sttCheckResult {
+func (s *Service) executeMySQLChecks(ctx context.Context, except []string) []sttCheckResult {
+	m := make(map[string]struct{}, len(except))
+	for _, e := range except {
+		m[e] = struct{}{}
+	}
+
 	checks := s.getMySQLChecks()
 
 	var res []sttCheckResult
 	for _, c := range checks {
+		if _, ok := m[c.Name]; ok {
+			continue
```

Comment:
should we add a log message here? "Skipping disabled check: ......" ?

Reply:
Added log message with `debug` level, as it's expected behavior and such log has value for the development.